Validation/RecoParticleFlow/plugins/BuildFile.xml

```diff
@@ -14,6 +14,7 @@
   <use name="CommonTools/UtilAlgos"/>
   <use name="Geometry/HcalTowerAlgo"/>
   <use name="MagneticField/Records"/>
+  <use name="FastSimulation/CaloGeometryTools"/>
```

Comment:
this doesn't seem like a dependency we want? In any case it looks like Transform3DPJ.h is just copied from root - perhaps the better solution is to depend on root directly;

https://github.com/cms-sw/cmssw/issues/26016

Reply:
Thanks David, I've implemented your suggestion